src/UiElement/UiElementTrait.php

```diff
@@ -23,6 +25,8 @@ trait UiElementTrait
 
     protected bool $ignored = false;
 
+    protected Environment $twig;
```

Comment:
**❔ question:** ‏What about having this nullable?

Reply:
Why should it be nullable ?